src/index.html

```diff
@@ -83,7 +83,7 @@
                 </div>
             </div>
         </section>
-        <section id="feedback" class="ui transition hidden container">
+        <section id="qrcoderesult" class="ui transition hidden container">
```

Comment:
Would use a `result` or `preview` ID here. Since it's will be displayed in the URL, simpler is better.